org.openhab.binding.zigbee.ember/src/main/java/org/openhab/binding/zigbee/ember/internal/discovery/ZigBeeEmberUsbSerialDiscoveryParticipant.java

```diff
@@ -0,0 +1,88 @@
+/**
+ * Copyright (c) 2014-2018 by the respective copyright holders.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.zigbee.ember.internal.discovery;
+
+import static java.util.Arrays.asList;
+import static org.openhab.binding.zigbee.ZigBeeBindingConstants.CONFIGURATION_BAUD;
+import static org.openhab.binding.zigbee.ZigBeeBindingConstants.CONFIGURATION_PORT;
+import static org.openhab.binding.zigbee.ember.EmberBindingConstants.THING_TYPE_EMBER;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import org.eclipse.jdt.annotation.Nullable;
+import org.eclipse.smarthome.config.discovery.DiscoveryResult;
+import org.eclipse.smarthome.config.discovery.DiscoveryResultBuilder;
+import org.eclipse.smarthome.config.discovery.usbserial.UsbSerialDeviceInformation;
+import org.eclipse.smarthome.config.discovery.usbserial.UsbSerialDiscoveryParticipant;
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+import org.eclipse.smarthome.core.thing.ThingUID;
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * Discovery for ZigBee USB dongles, integrated in Eclipse SmartHome's USB-serial discovery by implementing
+ * a component of type {@link UsbSerialDiscoveryParticipant}.
+ * <p/>
+ * Currently, this {@link UsbSerialDiscoveryParticipant} supports the ZigBee dongle 'BV AV2010/10' from Bitron Video.
+ *
+ * @author Henning Sudbrock - initial contribution
+ */
+@Component(service = UsbSerialDiscoveryParticipant.class)
+public class ZigBeeEmberUsbSerialDiscoveryParticipant implements UsbSerialDiscoveryParticipant {
+
+    private static final Set<ThingTypeUID> SUPPORTED_THING_TYPES = new HashSet<>(asList(THING_TYPE_EMBER));
+
+    public static final int SILICON_LABS_USB_VENDOR_ID = 0x10c4;
```

Comment:
just a minor comment - such constants have been documented for the Telegesis dongle

Reply:
ok